brad-chat/src/components/RagFileInput.js

```
@@ -18,6 +23,58 @@ function RagFileInput() {
         setDbName(e.target.value);  // Update the state when name input changes
     };
 
+    // Handlers for new Search Settings
+    const handleNumArticlesChange = async (e) => {
+        const newValue = e.target.value; // Get the updated value from the input field
+        setNumArticles(newValue); // Update the state with the new value
+    
+        // Ensure the newValue is a valid number before proceeding
+        if (!newValue || isNaN(newValue) || parseInt(newValue, 10) < 1) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)